src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj

```diff
@@ -3,6 +3,11 @@
     <ProjectGuid>{82B54697-0251-47A1-8546-FC507D0F3B08}</ProjectGuid>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <Configurations>netcoreapp-Debug;netcoreapp-Release;netstandard-Debug;netstandard-Release</Configurations>
+    <!-- 
+    We wish to test operations that would result in
+    "Operator '-' cannot be applied to operands of type 'ushort' and 'EnumArithmeticTests.UInt16Enum'"
```

Comment:
❔ Can you not use targeted warning suppressions to allow the specific cases?

Reply:
I could not get #pragma disable 0019 to work. Does it work for you?